modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/rest/api/SCIM2GroupTest.java

```diff
@@ -188,4 +209,163 @@ public void getErrorGroupIdPermissions() throws IOException {
                 .assertThat()
                 .statusCode(HttpStatus.SC_NOT_FOUND);
     }
+
+    @Test(dependsOnMethods = "testGETGroupDetails")
```

Comment:
Shall we add a description for the test?

Reply:
Resolved in https://github.com/wso2/product-is/pull/7439/commits/5f62c85aab90f9520469d3a7da2fb302dbdfcc27